src/main/java/DataStore/DataStore.java

```diff
@@ -1,5 +1,6 @@
 public interface DataStore {
 
+  public String listContent();
```

Comment:
The changes here aren't required for Cucumber tests to run, but they help isolate the unit tests from the acceptance tests. After creating the `Directory` class, I originally ran assertions using `http-server/src/test/resources/testFiles`. Since I know that I'll be adding files to this directory, I didn't want to have to change the expected results (particularly for `listContent()`.) every time I added a new file.